charts/backstage/Chart.yaml

```
@@ -38,4 +38,4 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.0.0
+version: 2.0.1
```

Can you please make this a feature release?

```suggestion
version: 2.1.0
```

This is not a fix, but a new value option - a new feature if you will. 🙂